index.js

```diff
@@ -36,7 +40,8 @@ function spawnClangFormat(args, done, stdio) {
     // This makes it impossible to format files called '-version' or '--version'. That's a feature.
     // minimist & Co don't support single dash args, which we need to match binary clang-format.
     console.log('clang-format NPM version', VERSION, 'at', LOCATION);
-    process.exit(0);
+    setImmediate(done);
```

Comment:
nit: can you add a comment that you're using `setImmediate` so that the calling code consistently gets async notifications?